frontend/src/views/WheresMyGeneV2/components/HeatMap/style.ts

```diff
@@ -161,16 +161,62 @@ export const YAxisWrapper = styled.div<YAxisWrapperProps>`
   overflow: hidden;
 `;
 
-export const StyledSkeleton = styled(Skeleton)`
-  margin: 0px 0px 4px 1px;
+interface LoadingContainerProps extends CommonThemeProps {
+  height: number;
+  width: number;
+}
+
+export const LoadingContainer = styled.div<LoadingContainerProps>`
+  position: absolute;
+  display: flex;
+  width: ${(props) => props.width * 20}px;
+  height: ${(props) => props.height * 20}px;
+  background-color: ${gray100};
 `;
 
-export const SkeletonContainer = styled.div`
+interface LoadingProps extends CommonThemeProps {
+  left: number;
+}
+
+export const LoadingWrapper = styled.div<LoadingProps>`
   display: flex;
   flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 600px;
+  left: ${(props) => ((props.left + 62) * 19) / 2}px;
 `;
 
-export const SkeletonWrapper = styled.div`
-  display: flex;
-  flex-direction: row;
+interface LoadingLabelProps extends CommonThemeProps {
+  visible: boolean;
+}
+
+export const LoadingLabel = styled.div<LoadingLabelProps>`
+  color: black;
+  ${fontHeaderXl}
+  padding-top: 20px;
+  visibility: ${(props) => (props.visible ? "visible" : "hidden")};
+`;
+
+export const LoadingSpinner = styled.div`
```

Comment:
SDS has a spinner  https://sds.czi.design/009eaf17b/p/429f42-loading-indicators/b/12a529 should we just use it instead?

Reply:
we could, but we won't be able to customize it to the design/requirements for this.

Reply:
Ah got it! What's the requirement that SDS spinner doesn't meet? Just wanted to double check the benefit is worth the ongoing maintenance cost of having a snowflake 😄 

CC: @hthomas-czi 

Thanks both!

Reply:
It's okay with me to use the SDS component, but I think it's too small and may be hard to notice. @tihuan It'd be nice if we added a `sdsSize` property to have a larger variant.

Reply:
That sounds great! I'll submit a ticket to SDS for this 👍 Keeping the current loader code for now SGTM!

Reply:
@tihuan After discussing with Ronen, we're planning to go with the custom loading indicator. When there are only a few genes being loaded, there isn't enough space for the entire LoadingIndicator, so we hide the text and just show the spinner. The SDS component doesn't support this behavior, but it'd be great to add to that ticket. Thanks!